docs/src/debugging.md

```diff
@@ -98,7 +98,9 @@ end
 
 FT = Float64
```

Comment:
I can't comment on that line, but there's a typo in line 42:
"If needed, `post_op_callback` can be specialized or behave differently in
different cases, but here, it only checks if `NaN`s are in the given that."
Maybe it should say "in the given object."?

Reply:
Also it might be helpful to give more details about what `post_op_callback` should look like, particularly what the arguments should be

Reply:
It's unfortunately difficult to generalize, because `post_op_callback` is called by many many different functions, so there is not an easy way to document what the arguments will be. I'll try to improve the docs on this.